.github/workflows/wheels.yml

```diff
@@ -84,16 +84,16 @@ jobs:
             cibw_skip: "pp* *musl*"
 
           # macOS x86
-          - os: macos-latest
+          # Note: keep as old as possible as due to libomp this will be the oldest supported macOS version.
+          - os: macos-12
             cibw_archs: "x86_64"
 
-          # macOS Apple Silicon cross-compiled on x86 macOS runner.
-          # GitHub does not offer Apple Silicon yet (only for self-hosted).
-          # See https://github.com/github/roadmap/issues/528
-          - os: macos-latest
+          # macOS Apple Silicon
+          # Note: keep as old as possible as due to libomp this will be the oldest supported macOS version.
+          - os: macos-14
             cibw_archs: "arm64"
-            # Skip macOS ARM tests on Intel runner.
-            cibw_test_skip: "*-macosx_arm64"
+            # Silence warning for only Python 3.8, as per https://github.com/pypa/cibuildwheel/pull/1169
+            cibw_test_skip: "cp38-macosx_*:arm64"
```

Comment:
If we're dropping Python 3.8 then this is no longer needed

Reply:
Done